colossalai/tensor/colo_tensor.py

```diff
@@ -31,21 +39,27 @@ def __new__(cls, data: torch.Tensor, spec: TensorSpec = TensorSpec(distspec.repl
         return torch.Tensor._make_subclass(cls, data, data.requires_grad)
 
     def __init__(self, data: torch.Tensor, spec: TensorSpec = TensorSpec(distspec.replicate())) -> None:
-        self._spec = copy(spec)
+        """__init__ of the tensor. It worth noting that the class's base class torch.Tensor 
```

Comment:
For our current API document render, the docstring of `__init__()` should be under the class instead of  `__init__()`  function.

Reply:
OK, fixed it!